### PR TITLE
Update to README

### PR DESCRIPTION
**Pull request by inudge**
Minor improvements to text

### CODE REVIEW

**Comment by inudge** [comment]
sorry, I should have wrote:

sudo brew services restart stubby

File: README.md [hunk 1]
```diff
@@ -1,16 +1,18 @@
...
 
 Stubby provides DNS Privacy by:
 
 * Running as a daemon
-* Listening on the loopback address to send all outgoing DNS queries received on that address out over TLS
+* Listening on the loopback address to send all outgoing DNS queries received on that address out over TLS (1.2 or 1.3)
 * Using a default configuration which provides Strict Privacy and uses a subset
 of the available [DNS Privacy servers](https://dnsprivacy.org/wiki/x/E4AT)
 
+Note: Resolution of DNS queries using TLS is typically several times slower than standard UDP based DNS, this may (but not necessarily) make a noticable difference to system performance.
```
Comment (wtoorop) [inline README.md]: Hi Inudge,

This is not necessarily true.  Once the TLS session is set up, the should not be a difference round trip-wise and could even be faster when multiple requests are pipe-lined (and talking to an upstream that supports Out Of Order responses)

**Comment by inudge** [comment]
Hi wtoorop,

Using round_robin_upstreams = 1 may be why I'm seeing slower resolution ?

**Comment by wtoorop** [comment]
It could be your RTT (latency) to the upstream servers.  But even then, performance might be better with stateful connections for certain query patterns (i.e. if you fill the pipe).

I am just 4 miliseconds from dnsovertls.sinodun.com (and less than 1 to getdnsapi.net), but I am located close to those servers. I guess you have to try out your latency to the different servers to find the ones which give you best performance.  I suppose dns.quad9.net might be your best bet in general (or if you travel a lot), since the address is anycasted across the globe.

Also, out of order responses are essential for good performance.  The sinodun.com servers provide that, but the getdnsapi.net not (yet).  (I don't know if quad9 does, but I'll try to test for that later)

**Comment by inudge** [comment]
Ok, I'll try that and let you know...

BTW I cannot use Quad9 servers due to problems with my ISP (probably routing errors but could be blocking). I have an open ticket for that with quad9 support.

**Comment by wtoorop** [comment]
Also, I suspect you have turned on DNSSEC validation (because of your issue).  This also reduces performance severely currently , because getdns does not yet have a cache.  Each query will currently cause all the queries to get to complete DNSSEC chain to be able to validate the response (with native Stubby DNSSEC validation).
A small cache (if only just for DNSSEC replies) is on the roadmap of getdns...

**Comment by inudge** [comment]
I get average RTTs of about 16 and 20 ms to dnsovertls.sinodun.com & getdnsapi.net respectively.

stubby -i | grep -i dnssec
[09:55:30.161194] STUBBY: Read config from file /usr/local/etc/stubby/stubby.yml
Result: Config file syntax is valid.
    "dnssec_allowed_skew": 0,
    "dnssec_return_all_statuses": GETDNS_EXTENSION_FALSE,
    "dnssec_return_full_validation_chain": GETDNS_EXTENSION_FALSE,
    "dnssec_return_only_secure": GETDNS_EXTENSION_FALSE,
    "dnssec_return_status": GETDNS_EXTENSION_FALSE,
    "dnssec_return_validation_chain": GETDNS_EXTENSION_FALSE,
    "dnssec_roadblock_avoidance": GETDNS_EXTENSION_FALSE,
    "trust_anchors_verify_email": <bindata of "dnssec@iana.org">,

**Comment by wtoorop** [comment]
Ok... that's not too bad.  So that tells me you're in Europe :).  You might want to try using the sinodun servers only... better RTT for you and they do out of order responses too.

**Comment by inudge** [comment]
Yep, Paris :)

I normally prefer spreading queries around for increased redundancy and privacy rather than maximising performance. However, it's good to know the best servers for when speed is the top priority, thanks !

**Pull request closed without merging**